provider/pkg/provider/provider.go

```diff
@@ -1629,7 +1654,6 @@ func (k *kubeProvider) Diff(ctx context.Context, req *pulumirpc.DiffRequest) (*p
 		Replaces:            replaces,
 		Stables:             []string{},
 		DeleteBeforeReplace: deleteBeforeReplace,
-		Diffs:               changes,
```

Comment:
Note: I got rid of the `Diffs` computation and am letting the engine handle this now. The `DetailedDiff` computation seems to be necessary to get an accurate diff with Server-side Apply. I tried setting `DiffResponse_DIFF_UNKNOWN`, but the result was incorrect.

Reply:
Interesting... Could you add a comment here in code to document the same? (why diffs are omitted?)

Reply:
FWIW, I think this might be a tiny compatibility break with some very old versions of the CLI (~v0.17.20, which was released in June 2019). I think that the only break would be in the display code, which may no longer render the list of changed properties. I think that this is acceptable.